recipes/lely-core/all/conanfile.py

```diff
@@ -0,0 +1,238 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import AutotoolsToolchain, Autotools
+from conan.tools.layout import basic_layout
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.files import (
+    export_conandata_patches,
+    apply_conandata_patches,
+    get,
+    copy,
+    rmdir,
+    rm,
+)
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.53.0"
+
+class LelyConan(ConanFile):
+    name = "lely-core"
+
+    # Optional metadata
+    license = "Apache"
+    homepage = "https://gitlab.com/lely_industries/lely-core/"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The Lely core libraries are a collection of C and C++ libraries and tools, providing hih-performance I/O and sensor/actuator control for robotics and IoT applications. The libraries are cross-platform and have few dependencies. They can be even be used on bare-metal microcontrollers with as little as 32 kB RAM."
```

Comment:
I know it's like this upstream too, but I've found a typo (Unless hih actually means something in the domain 🤷‍♂️!):
```suggestion
    description = "The Lely core libraries are a collection of C and C++ libraries and tools, providing high-performance I/O and sensor/actuator control for robotics and IoT applications. The libraries are cross-platform and have few dependencies. They can be even be used on bare-metal microcontrollers with as little as 32 kB RAM."
```

Reply:
yeah, i think that's just a typo